src/Repository/ProgrammeRepository.php

```diff
@@ -99,8 +110,9 @@ public function getPaginatedFilteredSorted(
             $direction = 'ASC';
         }
 
-        if ('' != $sort) {
-            $query = $query->orderBy("p.$sort", $direction);
+        if (null != $sort) {
+            $query
```

Comment:
Can be written on a single line

Reply:
The if statement or the code inside of it?

Reply:
The method call